packages/strapi-helper-plugin/lib/internals/testing/strapi.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * 
+ * Strapi
+ * This fle
+ * 
+ */
+
+
+// Setup the strapi functioon global variable
+
+const React = require('react');
+const hoistNonReactStatics = require('hoist-non-react-statics');
+
+const hoc = () => (WrappedComponent) => {
```

Comment:
the injectHook, registerPlugin should also be added to this file it will allow easier mocking